public/video-ui/src/components/Videos/VideoItem.js

```diff
@@ -40,29 +41,56 @@ export default class VideoItem extends React.Component {
 
   render() {
     const video = this.props.video;
-    const scheduledLaunch = video.contentChangeDetails.scheduledLaunch && video.contentChangeDetails.scheduledLaunch.date;
+    const scheduledLaunch =
```

Comment:
could we use this to make things DRYer? https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/util/video.js#L96